scimax-contacts.el

```diff
@@ -342,7 +342,7 @@ Optional argument PATH is ignored."
 
 
 ;; * Capture contacts in messages
-;; I use this this mu4e, but it should work in any message.
+;; I use this with mu4e, but it should work in any message.
```

Comment:
As mentioned earlier, in some cases I tried to pick other fixes. I'm not sure what was intended here, but it seems like there was a goal to have more than just `this`.